dsrag/knowledge_base.py

```diff
@@ -185,6 +187,7 @@ def add_document(
         Inputs:
         - doc_id: unique identifier for the document; a file name or path is a good choice
         - text: the full text of the document
+        - file_path: the path to the file containing the document text (if text is not provided)
```

Comment:
Need to specify which file types are supported. Also, I wouldn't say "document text" here, because it makes it sound like it needs to be a text file or something similar.